### DISCUSSION TITLE
API strangeness: 404 Errors on POST requests but 200 on GET

### DISCUSSION THREAD

**Discussion by MConley2g** [microsoft/AzUrlShortener#503] [Q&A]
Hi.
I just set this up yesterday. I followed the guide here: https://github.com/microsoft/AzUrlShortener/wiki/How-to-deploy-your-AzUrlShortener (skipping installation of TinyBlazorAdmin) but I'm running into a strange error:
If I try any of the GET requests (like GET {{BaseUrl}}/api/UrlList) - Everything works as it should.
But any of the POST requests (like POST {{BaseUrl}}/api/UrlUpsert) - I get a 404 response.

I can manually add urls to the table storage and they will show up with the get request and the links do work. But for some reason the POSTs are not accessable.

Can anybody point out what I am missing here?
Thanks in advance :)
-Mikael

**Comment by fboucher** [comment]
@MConley2g can you share more details of your POST call? What are you using, what JSON are you passing, etc.
The more the better :) 

Also I'm assuming it was a typo, can you confirm your POST is on the function **UrlCreate**?

**Comment by udaypat** [comment]
Seems to be a typo in the Postman collection. https://github.com/microsoft/AzUrlShortener/pull/587
